app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const {
+  getTopics,
+} = require("./controllers/app.controller");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
Is this necessary yet? What would you want `app.use(express.json())` for?  Please remove until needed